infarstructure/src/main/java/util/InfraUtil.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.stream.JsonReader;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class InfraUtil {
+
+    // jsonString to JSON object
+    public static JsonObject stringToJson(String jsonString) {
```

Comment:
Let's add here a `Class<T>` parameter. 
```suggestion
    public static JsonObject stringToJson(String jsonString, Class<T> tClass) {
```

If a user doesn't have a specific class to parse from, use `stringToJson(jsonString, Map.class)`